pkg/notify/notifier/wechat/wechat.go

```diff
@@ -21,34 +21,67 @@ import (
 )
 
 const (
-	DefaultApiURL           = "https://qyapi.weixin.qq.com/cgi-bin/"
-	DefaultSendTimeout      = time.Second * 3
-	ToUserBatchSize         = 1000
-	ToPartyBatchSize        = 100
-	ToTagBatchSize          = 100
-	AccessTokenInvalid      = 42001
-	DefaultTextTemplate     = `{{ template "nm.default.text" . }}`
-	DefaultMarkdownTemplate = `{{ template "nm.default.markdown" . }}`
-	MessageMaxSize          = 2048
-	DefaultExpires          = time.Hour * 2
+	DefaultApiURL                 = "https://qyapi.weixin.qq.com/cgi-bin/"
+	DefaultSendTimeout            = time.Second * 3
+	ToUserBatchSize               = 1000
+	ToPartyBatchSize              = 100
+	ToTagBatchSize                = 100
+	AccessTokenInvalid            = 42001
+	DefaultTextTemplate           = `{{ template "nm.default.text" . }}`
+	DefaultMarkdownTemplate       = `{{ template "nm.default.markdown" . }}`
+	MessageMaxSize                = 2048
+	DefaultExpires                = time.Hour * 2
+	ChatbotMessageMaxTextSize     = 2048
+	ChatbotMessageMaxMarkdownSize = 4096
 )
 
 type Notifier struct {
-	notifierCtl    *controller.Controller
-	receiver       *wechat.Receiver
-	accessToken    string
-	timeout        time.Duration
-	logger         log.Logger
-	tmpl           *template.Template
-	ats            *notifier.AccessTokenService
-	messageMaxSize int
-	tokenExpires   time.Duration
+	notifierCtl        *controller.Controller
+	receiver           *wechat.Receiver
+	accessToken        string
+	timeout            time.Duration
+	logger             log.Logger
+	tmpl               *template.Template
+	ats                *notifier.AccessTokenService
+	messageMaxSize     int
+	tokenExpires       time.Duration
+	chatbotThreshold   int
+	chatbotUnit        time.Duration
+	chatbotMaxWaitTime time.Duration
 }
 
 type weChatMessageContent struct {
 	Content string `json:"content"`
 }
 
+type text struct {
```

Comment:
This can be replaced by expanding `weChatMessageContent`.

Reply:
The text type of the robot needs a json structure, if I extend the `weChatMessageContent` structure, will it affect wechat?

Reply:
No